short/index.html

```diff
@@ -298,6 +298,35 @@
 </section>
 
 <section data-markdown>
+**Exercise**
+
+```python
+def test_overlay_no_overlap():
+    rect1 = Rectangle(0, 0, 1, 1)
```

Comment:
I know the right way to do this is with classes, but I always use flat lists [x0, y0, x1, y1] to reduce cognitive load.